operator/roles/forkliftcontroller/tasks/main.yml

```
@@ -235,6 +235,38 @@
       state: present
       definition: "{{ lookup('template', 'controller/imagestream-vddk.yml.j2') }}"
 
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)